app/models/course.rb

```diff
@@ -256,6 +256,8 @@ def self.entry_requirement_options_without_nil_choice
 
   after_update :send_notification_to_accredited_body, if: :notify_accredited_body?
 
+  after_create :send_create_notification_to_accredited_body, if: :notify_accredited_body?
```

Comment:
I don't think we should send these emails from callbacks here for a few reasons:

* the model should be concerned with persistence and data not email
* the effect of this is unexpected `Course.create` sends an email. Every time we want to create a course in tests we have to be aware of (and possibly stub) unintended consequences
* we've just added another 100 lines to the ~2.5k lines of spec for this class

I don't think this should block this PR but if everyone agrees I'll add another ticket to refactor this